torch/_inductor/lowering.py

```diff
@@ -6229,14 +6229,6 @@ def div_prim(a, b):
     if is_integral:
         return truncdiv(a, b)
 
-    if (divisor := get_constant_value(b)) is not None:
```

Comment:
Okay, sorry, per @ngimel, let's do you your original approach and disable this opt only on cpu